connectivity/drivers/lora/COMPONENT_SX126X/SX126X_LoRaRadio.cpp

```diff
@@ -703,7 +708,14 @@ uint8_t SX126X_LoRaRadio::get_device_variant(void)
         return SX1261;
     }
 }
+#else
+uint8_t SX126X_LoRaRadio::get_device_variant(void)
+{
+    return MBED_CONF_SX126X_LORA_DRIVER_DEVICE_VARIANT;
```

Comment:
this can be moved into the function as it was.

If `MBED_CONF_SX126X_LORA_DRIVER_DEVICE_VARIANT == -1`, we use the a value read via a pin, otherwise we use configurable value. Same function, just what it returns changes.